app/services.js

```diff
@@ -0,0 +1,58 @@
+const broadcast = new BroadcastChannel('/api/drive.v1');
+
+self.addEventListener('install', (event) => {
+  console.log('Install JupyterLite service v1');
```

Comment:
Should we keep these `console.log()` here? (some site deployers want quiet apps)